internal/index/index.go

```diff
@@ -187,10 +188,9 @@ func loadIndexJSON(ctx context.Context, repo ListLoader, id restic.ID) (*indexJS
 }
 
 // Load creates an index by loading all index files from the repo.
-func Load(ctx context.Context, repo ListLoader, p *restic.Progress) (*Index, error) {
+func Load(ctx context.Context, repo ListLoader, p *progress.Counter) (*Index, error) {
 	debug.Log("loading indexes")
 
-	p.Start()
 	defer p.Done()
```

Comment:
As the progress bar is now started outside`Load` I'd also like to move `p.Done()` to the calling site. The way the rule for stopping a progress bar stays the same as before: Whoever calls `Start()` (now done implicitly) also has to call `Done()`.